README.md

```diff
@@ -21,14 +104,26 @@ The structure of this repository is designed to be as simple as possible. The ma
   - `index.qmd` - The main file of the abstract.
   - `references.bib` - The BibTeX file for the references.
 
-## Usage
+### Previewing the Abstract
+
+Previewing your document ensures it looks correct before submission. Here is how you can preview your abstract using the Quarto CLI:
+
+1. **Install Quarto:**
+   - Follow the detailed installation instructions on the [Quarto website](https://quarto.org/docs/get-started/).
+
+2. **Open Your Terminal:**
+   - Access your terminal or command prompt. On Windows, you can search for "cmd" or "Command Prompt" in your start menu. On macOS, open "Terminal" from your Applications/Utilities folder.
+
+3. **Navigate to Your Repository:**
+   - Use the `cd` command to change directories to the root directory of your repository, where your `index.qmd` file is located.
```

Comment:
Should be root of repo, right?

Reply:
```suggestion
   - Use the `cd` command to change directories to the directory of your repository where your `index.qmd` file is located.
```

Reply:
```suggestion
   - Use the `cd` command to change directories to the directory of your repository where your `index.qmd` file is located.
```

Reply:
ATM it is, but not necessarily. See suggested change.